web/src/apps/main/redux/actions/articles-page/index.ts

```diff
@@ -61,7 +61,7 @@ export const fetchCurrentArticleContributors = (): ThunkResult<
   if (!currentArticle || Array.isArray(currentArticle.contributors)) return;
 
   const response = await Axios.get<GithubUser[]>(
-    apiURL + `/contributors?articleSlug=${currentArticle.slug}`,
+    apiURL + `/v2/contributors?articleSlug=${currentArticle.slug}`,
```

Comment:
as you can see in [here](https://github.com/dzcode-io/dzcode.io/blob/f54895bba3a432754d9174a4259e2c7999b4e61a/api/src/contributor/controller.ts#L27), the endpoint has changed a bit, now you should use it like this:

```suggestion
    apiURL + `/v2/contributors?path=articles/${currentArticle.slug}`,
```